### PR TITLE
global: code style, copyrights, authors

### PR DESCRIPTION
**Pull request by jirikuncar**
Signed-off-by: Jiri Kuncar <jiri.kuncar@cern.ch>

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -82,5 +84,4 @@ Jinja utilities for Invenio.
 - Valkyrie Savage <vasavage@gmail.com>
 - Yannick Tapparel <yannick.tapparel@cern.ch>
 - Yoan Blanc <yoan.blanc@cern.ch>
-- Kamil Neczaj <kneczaj@gmail.com>
```
Comment (tiborsimko) [inline AUTHORS.rst]: There are some commits from Kamil in the git history...

File: AUTHORS.rst [hunk 2]
```diff
@@ -47,7 +48,7 @@ Jinja utilities for Invenio.
 - Jaime García <jaime.garcia.llopis@gmail.com>
 - Jan Aage Lavik <jan.age.lavik@cern.ch>
 - Jan Iwaszkiewicz <jan.iwaszkiewicz@cern.ch>
-- Javier Martin <javier.martin.montull@cern.ch>
```
Comment (tiborsimko) [inline AUTHORS.rst]: There are some commits from Javier Martin Montull in the git history...

File: AUTHORS.rst (hunk 1 above)
Comment (jirikuncar) [inline AUTHORS.rst]: He is twice in the author list.

File: AUTHORS.rst (hunk 2 above)
Comment (jirikuncar) [inline AUTHORS.rst]: He is twice in the author list.

File: AUTHORS.rst (hunk 1 above)
Comment (tiborsimko) [inline AUTHORS.rst]: Ah, ok. :wink:

**Pull request merged**